packages/amplication-server/src/core/build/build.controller.ts

```diff
@@ -58,9 +57,9 @@ export class BuildController {
     EnvironmentVariables.instance.get(Env.CODE_GENERATION_FAILURE_TOPIC, true)
   )
   async onCodeGenerationFailure(
-    @Payload() message: KafkaMessage
+    @Payload() message: CodeGenerationSuccess
   ): Promise<void> {
-    const args = plainToInstance(CodeGenerationSuccess, message.value);
+    const args = plainToInstance(CodeGenerationSuccess, message);
```

Comment:
is `plainToInstance` still needed? 
Nestjs will attempt to parse the message to JSON and the right type here